README.md

```diff
@@ -7,3 +7,33 @@ This repo contains end-to-end tests that aim to cover the "Becoming a Teacher" s
 1. Clone this repo.
 2. `bundle install`
 3. `rake`
+
+## Work undertaken
+
+The intention of this repo was to explore end-to-end testing across the following BAT services:
+- [Publish](https://qa.publish-teacher-training-courses.service.gov.uk)
+- [Find](https://qa.find-postgraduate-teacher-training.service.gov.uk/)
+- [Apply](https://qa.apply-for-teacher-training.service.gov.uk/)
+- [Manage](https://qa.apply-for-teacher-training.service.gov.uk/)
+
+The goal was to provide testing to ensure that a course (once published), could be found by a candidate, applied for by a candidate and finally managed by a provider.
+
+Whilst we were able to write tests to met the needs of following this journey. We encountered several issues along the way, which prevented us from continuing this project at this time.
+
+## Encountered Issues
+
+1. To test these services we ran our scenarios against the QA environments of each service. Whilst some QA environments allowed us to sign in using personas, others required us to setup and sign in with DfE accounts. This required us to use notify to locate and acquire emails and links which would allow us to access the specific service.
```

Comment:
Despite looking similar, it actually wasn't DfE Sign-in. The Apply app has magic link logins powered by Devise for candidates – so completely standalone.

```suggestion
1. To test these services we ran our scenarios against the QA environments of each service. While the Publish QA environment allowed us to sign in using personas, Apply used 'magic links' for candidate authentication. This required us to use the GOV.UK Notify API to extract the magic links from sent emails which would allow us to sign in successfully.
```

Reply:
resolved in [1485593](https://github.com/DFE-Digital/bat-e2e-tests/pull/6/commits/1485593b2c99dfa0c3f7b2b5354843dd85efc273)